index.html.in

```diff
@@ -2,6 +2,7 @@
 
 <html>
     <head>
+        <meta name="theme-color" content="#2196F3"/>
```

Comment:
How did you pick this color?

Reply:
I selected this color as the Google Developers recommended color; although, I'd be happy to switch it to any other one.  Choosing a color is necessary to pass the "Address bar matches brand colors" test.

Reply:
Ah, I think this is fine.